src/Raven.Server/Documents/Queries/Results/QueryResultRetrieverBase.cs

```diff
@@ -177,6 +177,13 @@ protected Document GetProjection(Lucene.Net.Documents.Document input, Lucene.Net
 
                 if (doc == null)
                 {
+                    if (documentLoaded)
```

Comment:
What will happen if 1st field is projected from index and 2nd from document?

Reply:
The result will be skipped